language/e2e-tests/src/account.rs

```diff
@@ -208,9 +208,12 @@ impl Account {
         gas_currency_code: String,
     ) -> RawTransaction {
         match payload {
-            TransactionPayload::WriteSet(writeset) => {
+            TransactionPayload::WriteSet(WriteSetPayload::Direct(writeset)) => {
```

Comment:
recording for posterity that I think we should change the name from AdminScript to WritesetScript to reinforce that these are subsets of the existing writeset functionality.